RandomNumber.py

```diff
@@ -0,0 +1,35 @@
+import random
+
+def play_game():
+    number = random.randint(1, 100)
+    max_attempts = 5
+    attempts = 0
+
```

Comment:
اینجا max_attempts یک ثابته
بهتره که constant ها رو بیرون تابع و پایین import ها بذاری که بشه اونا رو از فایل های دیگه ایمپورت کرد اگر لازم بود.
```suggestion
MAX_ATTEMPTS = 5

def play_game(max_attempts=MAX_ATTEMPTS):
    number = random.randint(1, 100)
    attempts = 0

```